src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Commands/DefaultCommandHandler.cs

```diff
@@ -145,7 +145,13 @@ private static object[] GetInvokeParametersForMethod(MethodInfo methodInfo, ILis
                 }
             }
 
-            if (methodHasParams && (methodParameters.Length - args.Count == 1)) invokeParameters.Add(new string[] { });
+            var lastParameterIsParams =
+                methodParameters.Last().GetCustomAttributes(typeof(ParamArrayAttribute), false).Length > 0;
+
+            if (methodHasParams && (methodParameters.Length - args.Count == 1) && !lastParameterIsParams)
+            {
+                invokeParameters.Add(new string[] { });
+            }
```

Comment:
I'm really not sure why this logic was there. However, adding a second `string[]` argument to a method invocation that has a `params` parameter (in addition to an already added `string[]` argument on line 131) seems like a no-go in any case.